libraries/AP_Camera/examples/send_camera_information.py

```diff
@@ -0,0 +1,106 @@
+import threading
+import time
+from pymavlink import mavutil
+
+ONBOARD_CONTROLLER_SYSID = 245
```

Comment:
I think we should accept a command line argument for the system id and component id.  If those arguments aren't specified then I think we should consume HEARTBEAT messages and use the first system id that we receive as long as the MAV_TYPE (of the HEARTBEAT message) is not a GCS.  For component id we should probably default to using MAV_COMP_ID_ONBOARD_COMPUTER (191)